acme/setup.py

```diff
@@ -12,6 +12,7 @@
     'cryptography>=0.8',
     # Connection.set_tlsext_host_name (>=0.13), X509Req.get_extensions (>=0.15)
     'PyOpenSSL>=0.15',
+    'dnspython',
```

Comment:
I'll have a look at that (new to python).

I see your point, but I feel it's a little odd to have something covered by the core standard require an "extras" library.  And given the core standard includes DNS queries some DNS library dependency is to be expected.